install/empty_data.php

```diff
@@ -656,6 +656,675 @@
    ],
 ];
 
+
+$tables['glpi_dashboards_dashboards'] = [
+   [
+      "id"      => 1,
+      "key"     => "central",
+      "name"    => "Central",
+      "context" => "core",
+   ], [
+      "id"      => 2,
+      "key"     => "assets",
+      "name"    => "Parc",
+      "context" => "core",
+   ], [
+      "id"      => 3,
+      "key"     => "assistance",
+      "name"    => "Assistance",
+      "context" => "core",
+   ], [
+      "id"      => 4,
+      "key"     => "mini-tickets",
+      "name"    => "Mini tickets",
+      "context" => "mini_core",
+   ],
+];
+
+$tables['glpi_dashboards_items'] = [
```

Comment:
Do you think it could be factorized with the update script ?
Maybe using an `inlude` is possible (see https://www.php.net/manual/en/function.include.php#example-127 )